apps/studio/components/layouts/OrganizationLayout.tsx

```diff
@@ -91,7 +89,29 @@ const OrganizationLayout = ({ children }: PropsWithChildren<{}>) => {
           </NavMenu>
         </ScaffoldContainer>
       </ScaffoldHeader>
+
       <ScaffoldDivider />
+
+      {selectedOrganization && selectedOrganization?.managed_by !== 'supabase' && (
+        <ScaffoldContainer className="mt-8">
+          <Alert_Shadcn_ variant="default" className="flex items-center gap-4">
+            <PartnerIcon organization={selectedOrganization} showTooltip={false} size="medium" />
+            <AlertTitle_Shadcn_ className="flex-1">
+              This organization is managed by {PARTNER_TO_NAME[selectedOrganization.managed_by]}.
+            </AlertTitle_Shadcn_>
+            <Button type="default" iconRight={<ExternalLink />} asChild>
+              <a
+                href={`https://vercel.com/${'supabase'}/~/integrations/products/${process.env.NEXT_PUBLIC_VERCEL_MARKETPLACE_INTEGRATION_NAME}/${selectedOrganization.partner_id}`}
```

Comment:
nit: unnecessary interpolation but again not sure if this was intended based on this [comment](https://github.com/supabase/supabase/pull/28786/files#r1724634991)